### ISSUE TITLE
[Site Request] cmoa.jp

### ISSUE THREAD

**Issue by ishidayuu** [manga-download/hakuneko#7571]
**Name of the website**
https://www.cmoa.jp/

**Website urls (examples below)**
- Site: https://www.cmoa.jp/

- Manga List: 
https://www.cmoa.jp/boyslove/

- Manga example: https://www.cmoa.jp/title/303666/

- Chapter online viewer example: https://www.cmoa.jp/bib/speedreader/?cid=0000303666_jp_0001&u0=1&u1=0&rurl=https%3A%2F%2Fwww.cmoa.jp%2Ftitle%2F303666%2F

**Languages**
Japanese

Please add it to Harukeno. Thank you

**Comment by MikeZeDev** [comment]
are you serious? 

https://github.com/manga-download/hakuneko/blob/55ae7ed57cef196933890824ff5068df18f93e49/src/web/mjs/connectors/Cmoa.mjs#L4

**Comment by ishidayuu** [comment]
Thank you for your reply. Sorry, I don't know English and have been using a translator to read. I also don't understand programming. I just started using Haruneko, and I’ll continue exploring how to use it.

**Comment by ishidayuu** [comment]
> are you serious?
> 
> https://github.com/manga-download/hakuneko/blob/55ae7ed57cef196933890824ff5068df18f93e49/src/web/mjs/connectors/Cmoa.mjs#L4



> are you serious? 你认真吗？
> 
> https://github.com/manga-download/hakuneko/blob/55ae7ed57cef196933890824ff5068df18f93e49/src/web/mjs/connectors/Cmoa.mjs#L4

Sorry, I can search for CMOA on Hakuneko, but I can’t search for CMOA on HARUNEKO. Previously, I wasn’t able to open the old version of Hakuneko. Later, I used ChatGPT to translate the tutorial, read it repeatedly, and followed the instructions, and finally managed to open it. 
However, it started to have many new issues. Since I don't understand English and everything is in English.I had to spend a long time figuring out what was going on.
But when I used HARUNEKO, it seemed like there were fewer issues, so I’m using HARUNEKO now. 
If I want to add CMOA on HARUNEKO, what should I do? Where should I put the file from the link: https://github.com/manga-download/hakuneko/blob/55ae7ed57cef196933890824ff5068df18f93e49/src/web/mjs/connectors/Cmoa.mjs?

Sorry, I know my question might sound silly.

**Comment by ishidayuu** [comment]
> are you serious? 你认真吗？
> 
> https://github.com/manga-download/hakuneko/blob/55ae7ed57cef196933890824ff5068df18f93e49/src/web/mjs/connectors/Cmoa.mjs#L4

Sorry, I can search for CMOA on Hakuneko, but I can’t search for CMOA on HARUNEKO. Previously, I wasn’t able to open the old version of Hakuneko. Later, I used ChatGPT to translate the tutorial, read it repeatedly, and followed the instructions, and finally managed to open it. 
However, it started to have many new issues. Since I don't understand English and everything is in English.I had to spend a long time figuring out what was going on.
But when I used HARUNEKO, it seemed like there were fewer issues, so I’m using HARUNEKO now. 
If I want to add CMOA on HARUNEKO, what should I do? Where should I put the file from the link: https://github.com/manga-download/hakuneko/blob/55ae7ed57cef196933890824ff5068df18f93e49/src/web/mjs/connectors/Cmoa.mjs?

Sorry, I know my question might sound silly.
